README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+## About Me
+HELLO! My name is **Jarret Angbazo** and I am a second-year MPP student at the *Harris School of Public Policy*.
+
+Quick two truths and a lie:
+
+1. I did this assignment in 5 minutes
+
```

Comment:
Perhaps these blank lines are unnecessary, maybe not. Could be something to experiment with stylistically.